spec/ParseQuery.spec.js

```diff
@@ -3850,7 +4141,6 @@ describe('Parse.Query testing', () => {
       })
       .then(function (savedFoobar) {
         const foobarQuery = new Parse.Query('Foobar');
-        foobarQuery.include('barBaz');
         foobarQuery.select(['fizz', 'barBaz.key']);
```

Comment:
What we really want to test is this code: https://github.com/parse-community/parse-server/blob/0f5928542c44fb8b72ddca4a2d35c0905e2d335d/src/RestQuery.js#L73-L98

Making sure that when `select` is used, the correct keys are included without specifying `include`

Reply:
Makes sense